bin/docker-setup.sh

```diff
@@ -31,6 +31,17 @@ while [[ $? -ne 0 ]]; do
 	cli wp db check --path=/var/www/html --quiet > /dev/null
 done
 
+# If the plugin is already active then return early
+cli wp plugin is-active woocommerce-payments > /dev/null
+if [[ $? -eq 0 ]]; then
```

Comment:
@marcinbot: `if [[ ]]` is bash-specific, AFAIK. Perhaps using `#!/bin/bash` instead of `#!/usr/bin/env sh` at the first line would nicely clarify which syntax and utils are available.

Reply:
Super, thanks!